quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/Configuration.java

```diff
@@ -141,6 +141,10 @@ public static Optional<Boolean> getOptionalBooleanKcValue(String propertyName) {
         return getOptionalValue(NS_KEYCLOAK_PREFIX.concat(propertyName)).map(Boolean::parseBoolean);
     }
 
+    public static Optional<Boolean> getOptionalBooleanKcValue(Option<Boolean> option) {
```

Comment:
We have `Configuration.isTrue(Option<Boolean> option)`, which should cover these cases.